packages/did-core-test-server/suites/did-spec/did-producer.js

```diff
@@ -0,0 +1,70 @@
+const utils = require('./utils');
+const jsonMediaTypes = ['application/did+ld+json', 'application/did+json'];
+const deepEqual = require('deep-equal')
+
+const generateDidProducerTests = ({did, resolutionResult}) => {
+  const {didDocument} = resolutionResult;
+  const contentType = resolutionResult.didDocumentMetaData['content-type'];
+
+  it('6.1 Production and Consumption - A conforming producer MUST take a ' +
+    'DID document data model and a representation-specific entries map as ' +
+    'input into the production process. The conforming producer MAY accept ' +
+    'additional options as input into the production process.', async () => {
+      if(jsonMediaTypes.includes('application/did+ld+json')) {
```

Comment:
```suggestion
      if(jsonMediaTypes.includes(contentType)) {
```

Is this what you meant?

Reply:
Good catch, no, I meant `contentType === 'application/did+ld+json'` ... fixed in c0fb4a72b515e5157bd7b2e3dca93d2f41aa7292.

The reason the test only tests 'application/did+ld+json' is because that's the only time where you know that there is a representation-specific entries map.